.travis.yml

```diff
@@ -30,6 +30,7 @@ before_script:
 
 script:
 - travis/deploy.bash
+- ssh-agent -k || true
```

Comment:
Is this necessary? Or did this slip in because we haven't rebased this branch recently?

I ask mostly because I remember Mike saying we had an issue with Travis and the ssh-agent running continuously, and I want to make sure we don't regress once this gets merged in.